### DISCUSSION TITLE
FXGL with Scene Builder

### DISCUSSION THREAD

**Discussion by lancebord** [AlmasB/FXGL#1167] [Q&A] [answered]
I know there has been a discussion about this, but it doesn't quite solve my issue, or at least I'm not sure how to use the answer provided to help myself. Anyways, I am currently wondering if there is a way to put an FXGL application window into a scene builder pane. I'm assuming it's possible just not sure how to do it. Thanks!

**Comment by AlmasB** [comment, marked as answer]
Hi, if you mean to do this through code, then yes, `FXGLPane` is just a normal JavaFX node. This would be the easiest approach: build a .fxml with Scene Builder, load it into a JavaFX application, inject (something like `desiredParent.getChildren().add(...)`) `FXGLPane` via the controller.

If you wish to do this via the Scene Builder UI, then probably no.

**Reply by lancebord** [reply]
that should work, thank you!
